testsuite/tests/libratest/smoke_test.rs

```diff
@@ -283,15 +283,19 @@ fn smoke_test_single_node() {
 
 #[test]
```

Comment:
chatted last night about this, but reminder if you could add a comment describing what this test is about